yt_dlp/postprocessor/ffmpeg.py

```diff
@@ -659,12 +661,35 @@ def run(self, info):
 
         temp_filename = prepend_extension(filename, 'temp')
         self.to_screen('Embedding subtitles in "%s"' % filename)
-        self.run_ffmpeg_multiple_files(input_files, temp_filename, opts)
-        os.replace(temp_filename, filename)
+        if info['ext'] in self.SUPPORTS_LYRICS:
+            self.embed_lyrics(info['filepath'], sub_dict=info['requested_subtitles'])
+        else:
+            self.run_ffmpeg_multiple_files(input_files, temp_filename, opts)
+            os.replace(temp_filename, filename)
 
         files_to_delete = [] if self._already_have_subtitle else sub_filenames
         return files_to_delete, info
 
+    def embed_lyrics(self, audio_file, sub_dict):
+        if len(sub_dict) > 1:
+            self.report_warning('More than one subtitle file found. Your media player will likely be unable to display all of them.')
+
+        if sub_dict[list(sub_dict.keys())[0]]['ext'] != 'lrc':
+            raise PostProcessingError('LRC subtitles required. Use "--convert-subs lrc" to convert')
+
+        lyrics_list = []
+        for lyrics in sub_dict.keys():
+            lyrics_list.append(sub_dict[lyrics]['data'])
+        if audio_file.endswith('.mp3'):
+            for lyrics in lyrics_list:
+                audio = mutagen.id3.ID3(audio_file)
+                audio.add(mutagen.id3.USLT(encoding=3, lang='und', desc='', text=lyrics))
+                audio.save()
```

Comment:
```suggestion
            audio = mutagen.id3.ID3(audio_file)
            for lyrics in lyrics_list:
                audio.add(mutagen.id3.USLT(encoding=mutagen.id3.Encoding.UTF8, lang='und', text=lyrics))
            audio.save()
```
We can leave the description, and we want to write it only once preferably. And again, using `mutagen.id3.Encoding.UTF8` over just `3`?